src/Core/Settings.cpp

```diff
@@ -4410,7 +4410,7 @@ Possible values:
 
 Cloud default value: `none`.
 )", 0) \
-    DECLARE(UInt64, distributed_ddl_entry_format_version, 5, R"(
+    DECLARE(UInt64, distributed_ddl_entry_format_version, 8, R"(
```

Comment:
This will break rolling upgrades and will make downgrades impossible

Reply:
Keep the default value as is. Upgrade. Set it to 8 in config when you are sure that you will not downgrade. Update the default value after the next LTS, add an upgrade note to changelog.